commands/displayers/droplet_backup_policies.go

```diff
@@ -0,0 +1,50 @@
+package displayers
+
+import (
+	"io"
+
+	"github.com/digitalocean/doctl/do"
+)
+
+type DropletBackupPolicy struct {
+	DropletBackupPolicies []do.DropletBackupPolicy
+}
+
+var _ Displayable = &DropletBackupPolicy{}
+
+func (d *DropletBackupPolicy) JSON(out io.Writer) error {
+	return writeJSON(d.DropletBackupPolicies, out)
+}
+
+func (d *DropletBackupPolicy) Cols() []string {
+	cols := []string{
+		"DropletID", "BackupEnabled", "BackupPolicyPlan", "BackupPolicyWeekday", "BackupPolicyHour",
+		"BackupPolicyWindowLengthHours", "BackupPolicyRetentionPeriodDays",
+		"NextBackupWindowStart", "NextBackupWindowEnd",
+	}
+	return cols
+}
+
+func (d *DropletBackupPolicy) ColMap() map[string]string {
+	return map[string]string{
+		"DropletID": "Droplet ID", "BackupEnabled": "Backup Enabled",
+		"BackupPolicyPlan": "Backup Policy Plan", "BackupPolicyWeekday": "Backup Policy Weekday", "BackupPolicyHour": "Backup Policy Hour",
+		"BackupPolicyWindowLengthHours": "Backup Policy Window Length Hours", "BackupPolicyRetentionPeriodDays": "Backup Policy Retention Period Days",
+		"NextBackupWindowStart": "Next Backup Window Start", "NextBackupWindowEnd": "Next Backup Window End",
```

Comment:
The output of these commands is very wide. It might make sense to remove the redundant `BackupPolicy` labels. Since the output is in the context of listing or getting a backup policy, I think it would still be clear. Something like:

```suggestion
		"DropletID": "Droplet ID", "Enabled": "Enabled",
		"Plan": "Plan", "Weekday": "Weekday", "Hour": "Hour",
		"WindowLengthHours": "Window Length Hours", "RetentionPeriodDays": "Retention Period Days",
		"NextWindowStart": "Next Window Start", "NextWindowEnd": "Next Window End",
```

What do you think?

Reply:
Thanks, Andrew. I'll update tomorrow.

Reply:
Updated in [3b4e2da](https://github.com/digitalocean/doctl/pull/1609/commits/3b4e2da6b2cb6688c1f26ae307194eca3c8e9f70)